liferay-workspace/tests/src/test/groovy/com/liferay/blade/tests/BladeTests.groovy

```diff
@@ -0,0 +1,134 @@
+package com.liferay.blade.tests
+
+import org.gradle.testkit.runner.GradleRunner
+import static org.gradle.testkit.runner.TaskOutcome.*
+import spock.lang.Specification
+import de.undercouch.gradle.tasks.download.Download;
+import java.util.jar.JarInputStream;
+import java.io.FileNotFoundException;
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
+import okhttp3.Request.Builder;
+
+class BladeTests extends Specification {
+	public void checkManifest(arg1='', arg2='', arg3='') {
+		def jarInputStream = new JarInputStream(new FileInputStream("${arg1}"))
+
+		jarInputStream.withCloseable {
+			if (jarInputStream.manifest.getMainAttributes().getValue("${arg2}") == "${arg3}" ) {
+				return true;
+			}
+			else {
+				return false;
+			}
+		}
+	}
+
+	public void executeBlade(arg1='', arg2='', arg3='', arg4='') {
```

Comment:
We need to make this variable arugments, because we may want to execute blade that has 2 arguments or 5 arguments, etc